VPLAY-9906 [Foxtel][AAMP]Crash is observed during 4K VOD playback in …

dp0000 opened the pull request:
…XiOne-Foxtel

Reason for change: Updating Curldownloader instances upon calling UpdateMaxDrmSession

Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: I2b4c22d9a381f792eb08dfe8a49eecabe6f374d6
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
VPLAY-9906 [Foxtel][AAMP]Crash is observed during 4K VOD playback in XiOne-Foxtel
Reason for change: Updating Curldownloader instances upon calling UpdateMaxDrmSession

Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: I2b4c22d9a381f792eb08dfe8a49eecabe6f374d6
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/aamp?pullRequest=184) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/aamp?pullRequest=184) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: psiva01<br/>:white_check_mark: pstroffolino<br/>:x: Deepikasri1234<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/aamp?pullRequest=184) it.</sub>

psiva01 committed:
VPLAY-9906 [Foxtel][AAMP]Crash is observed during 4K VOD playback in XiOne-Foxtel

Reason for change: Addressed review comments.
Test Procedure: Refer Ticket
Risks: Low

pstroffolino committed:
Merge branch 'dev_sprint_25_2' into _foxtelcrash
